system/settings/settings.xml

```diff
@@ -2249,7 +2253,7 @@
         </setting>
         <setting id="audiooutput.ac3transcode" type="boolean" parent="audiooutput.ac3passthrough" label="667" help="36429">
           <level>2</level>
-          <default>false</default>
+          <default>true</default>
```

Comment:
Can drop it from this PR if it requires discussion and move to follow as I was planning.